evaluate/metrics/assessment.go

```diff
@@ -137,28 +137,33 @@ func (a Assessments) StringCSV() (row []string) {
 }
 
 func csvHeader() []string {
-	return append([]string{"model", "score"}, allAssessmentKeysStrings...)
+	return append([]string{"model", "repository", "score"}, allAssessmentKeysStrings...)
 }
 
 // FormatStringCSV formats the given assessment metrics as CSV.
```

Comment:
Should we call this just `FormatCSV` it is the only one. so no need to give it a type